gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -331,6 +346,30 @@ gpii.flowManager.local.savePreferences = function (that, gpiiKey, preferences) {
     }
 };
 
+/**
+ * Calculate the schemas containing preferences with their the default values from the reset to default file.
+ * @param {Object} defaultSettings - The value of flowManager.options.defaultSettings.
+ * @param {Object} schemas - The value of ontologyHandler.ontologyMetadata.flat.
+ * @return {Object}  - A collected schema containing schemas for each preference defined in the reset to default file,
```

Comment:
Either a @typedef or an example would help the reader since it is not terribly clear from the description what the structure actually is